annotations/src/main/java/com/google/errorprone/annotations/RestrictedApi.java

```diff
@@ -103,7 +103,7 @@
    *
    * <p>This should only be used if callers should aggressively move away from this API (or change
    * to a allowlist annotation after review). Too many warnings will lead to ALL warnings being
-   * ignored, so tread very carefully.
+   * ignored, so treat very carefully.
```

Comment:
I think that the original was [meant to be that way](https://plainenglish.com/expressions/tread-carefully/) :)

Reply:
Reverted to the original version, sorry for the noise!